tests/test_claude_wrapper_hooks.py

```
@@ -645,7 +768,9 @@ def main() -> int:
     test_live_socket_preserves_claude_auth_for_resume_launch(failures)
     test_live_socket_preserves_only_listed_claude_auth_keys(failures)
     test_live_socket_enforces_heap_cap_for_space_separated_flag(failures)
-    test_live_socket_tmpdir_failure_skips_node_options_injection(failures)
+    test_live_socket_whitespace_cache_path_skips_node_options_injection(failures)
```

Ruff (0.15.12)

[error] Undefined name `test_live_socket_whitespace_cache_path_skips_node_options_injection`

(F821)